apps/web/src/styles/verify-purchase.css

```
@@ -299,34 +1268,40 @@
 
 .vp-oauth-verify-btn {
   display: inline-flex;
   align-items: center;
   gap: 0.45rem;
-  padding: 0.5rem 1.1rem;
+  padding: 0.45rem 0.95rem;
   border-radius: 999px;
   border: 1px solid var(--brand, rgba(255, 255, 255, 0.25));
   background: color-mix(in srgb, var(--brand, rgba(255, 255, 255, 0.1)) 20%, transparent);
   color: #ffffff;
   font-family: "AirbnbCereal", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "AirbnbCereal" (font-family-name-quotes)

(font-family-name-quotes)